lms/static/styles/components/_BookList.scss

```diff
@@ -0,0 +1,31 @@
+// Height of cover images, chosen to work when the file picker app is shown
```

Comment:
I'm not diving in on this styling in this iteration...it gets the job done fine for now, and, with the exception of the "jumpiness" of the book-cover container, is OK to land as-is and we'll tune it later.